### PR TITLE
Fix FILLED_MAP crash

### PR DESCRIPTION
**Pull request by dktapps**
Untested

### CODE REVIEW

**Comment by Aericio** [comment]
So that was a bug in CS? In the crashdump, why did it say it was caused by a plugin...

**Comment by dktapps** [comment]
@Aericio maybe a plugin tried Item::get(Item::FILLED_MAP)? No idea, but yes it's a core bug

File: src/pocketmine/item/Item.php [hunk 1]
```diff
@@ -785,7 +785,7 @@ public static function init(){
 			self::$list[self::POTATO] = Potato::class;
 			self::$list[self::BAKED_POTATO] = BakedPotato::class;
 			self::$list[self::POISONOUS_POTATO] = PoisonousPotato::class;
-			self::$list[self::FILLED_MAP] = FILLED_MAP::class;
+			self::$list[self::FILLED_MAP] = FilledMap::class;
```
Comment (inxomnyaa) [inline src/pocketmine/item/Item.php:788]: oh my

Comment (inxomnyaa) [reply src/pocketmine/item/Item.php:788]: We found our noob, @robske110 gg xD

**Pull request merged**